prow/tide/tide_test.go

```diff
@@ -1582,6 +1589,123 @@ func TestSync(t *testing.T) {
 	}
 }
 
+func TestIsPassing(t *testing.T) {
+	yes := true
+	no := false
+	headSHA := "head"
+	success := string(githubql.StatusStateSuccess)
+	failure := string(githubql.StatusStateFailure)
+	testCases := []struct {
+		name             string
+		passing          bool
+		optional         []string
+		config           *config.Policy
+		combinedContexts map[string]string
+	}{
+		{
+			name:    "required checks disabled should not impact - passing",
```

Comment:
these names are a bit confusing

Reply:
thanks for pointing that out. After reviewing some of the test I discovered some issues.